app/i18n/global-messages.js

```diff
@@ -132,6 +132,10 @@ const globalMessages = defineMessages({
     id: 'settings.support.faq.faqLinkURL',
     defaultMessage: '!!!https://yoroi-wallet.com/faq/',
   },
+  twitterLinkUrl: {
```

Comment:
why did this change?

Reply:
To get the link during ServerErrorBanner.js